scripts/battlefields/Bearclaw_Pinnacle/flames_for_the_dead.lua

```diff
@@ -0,0 +1,34 @@
+-----------------------------------
+-- Flames for the Dead
+-- Bearclaw Pinnacle mission battlefield
+-- !pos -720 9 -441
+-----------------------------------
+require("scripts/globals/battlefield")
+require("scripts/globals/keyitems")
+require("scripts/globals/titles")
+require("scripts/globals/missions")
+require("scripts/globals/zone")
+-----------------------------------
+
+local content = BattlefieldMission:new({
+    zoneId        = xi.zone.BEARCLAW_PINNACLE,
+    battlefieldId = xi.battlefield.id.FLAMES_FOR_THE_DEAD,
+    maxPlayers    = 6,
+    levelCap      = 99,
+    timeLimit     = utils.minutes(30),
+    index         = 0,
+    entryNpc      = "Wind_Pillar_1",
+    exitNpc       = "Wind_Pillar_Exit",
+
+    missionArea           = xi.mission.log_id.COP,
+    mission               = xi.mission.id.cop.THREE_PATHS,
+    missionStatusArea     = xi.mission.log_id.COP,
+    missionStatus         = xi.mission.status.ULMIA,
+    requiredMissionStatus = 8,
+    title                 = xi.title.TRUE_COMPANION_OF_LOUVERANCE,
+    grantXP               = 1000,
+})
```

Comment:
👍

Reply:
Are there any examples where missionArea and missionStatusArea are different?  This might be redundant

Reply:
That's something I'm not 100% sure about. I believe they are different for the shared starter city missions such as The Shadow Lord Fight but perhaps I'm wrong. I haven't really worked on that code so I'm a little uncertain. If they are redundant then I can remove missionStatusArea.